SampleApp/SampleApp.csproj

```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Discord.Net.Commands" Version="1.0.1" />
-    <PackageReference Include="Discord.Net.WebSocket" Version="1.0.1" />
+    <PackageReference Include="Discord.Net.Commands" Version="1.0.2" />
```

Comment:
This update should be noted. Is this really necessary?

Reply:
Yes, it is. The 1.0.1 version has some bug and result is following exception 
`System.InvalidOperationException: Unknown guild channel type`.